home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/UserServiceImpl.java

```diff
@@ -87,6 +84,17 @@ public UserDto updateUser(Long id, UserDto updateUserDto) {
         return mapper.convert(fromDB, UserDto.class);
     }
 
+    private User checkingUniqueEmailParameterForUpdate(Long id, UserDto userDto) {
+        var userFindById = userRepository.findById(id).filter(User::getEnabled)
```

Comment:
I think you have to revork this method in way of email uniqueness validation without returning a user

Reply:
This method was chosen to reduce the number of duplicate lines. I have to check check multiple event options. And pulling it out of the user base when checking and updating is superfluous. 
If you have any suggestions on how to do this, I will take note of this.